src/components/individual-transcript/TranscriptTabContent.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { SetStateAction } from "react";
+import MarkdownPreview from "@uiw/react-markdown-preview";
+import "./markdown.css";
+import { InView } from "react-intersection-observer";
+import { createContentSlug } from "@/utils";
+
+function formatSpeakerText(text: string): string {
+  // Regular expression pattern for "Speaker: HH:MM:SS"
+  const pattern = /(\b\w+(?: \w+)*):? (\d{2}:\d{2}:\d{2})/g;
```

Comment:
We might have occurrences where the timestamp is missing. Can we also handle the `"Speaker:"` followed by new line case?

Reply:
Yh, we can but it won't still handle cases like this.
[Chaincode-Podcast](http://btctranscripts.com/chaincode-podcast/2020-11-30-carl-dong-reproducible-builds)

Reply:
tried to handle this but it catches some other things like https: if there's a space. what do you suggest?

Reply:
> Yh, we can but it won't still handle cases like this.

It's fine if we don't catch those cases.